internal/dag/httpproxy_processor.go

```diff
@@ -660,6 +668,11 @@ func (p *HTTPProxyProcessor) computeRoutes(
 				DNSLookupFamily:       string(p.DNSLookupFamily),
 				ClientCertificate:     clientCertSecret,
 			}
+
+			if len(p.MinimumTLSVersion) != 0 {
```

Comment:
here and other places, what if `CipherSuites` is set but not `MinimumTLSVersion`? seems like we would not get the ciphers set properly?

Reply:
seems like we would need separate checks for those two things being set

Reply:
Yes, I think @sunjayBhatia is right here, users should be able to set the minimum TLS version and ciphersuites separately.

Reply:
Ok. I will make the change accordingly. Would it make sense to add 2 separate flags to enable this feature in that case?
EnableUpstreamTLSVersion and EnableUpstreamTLSCipherSuites?

Reply:
I don't think we need two separate flags; we should just check each variable independently in the code.